tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend.robot

```diff
@@ -53,6 +53,18 @@ Upload subject
     user checks headed table body row contains  Data file size   17 Kb  ${section}
     user checks headed table body row contains  Status           Complete  ${section}  180
 
+Add meta guidance to subject
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until h2 is visible  Public metadata guidance document
+
+    user waits until page contains accordion section  Dates test subject
+    user opens accordion section  Dates test subject
+    ${text_element}=  user gets meta guidance accordion text element  Dates test subject
```

Comment:
Given the suggest keyword change, this would be something like `${editor}` instead